README.md

```diff
@@ -18,7 +18,7 @@ Sample Performance Tests - Banking (Using [JUnit](https://github.com/junit-team/
 ```xml
 <dependency>
     <groupId>org.jsmart</groupId>
-    <artifactId>zerocode-tdd</artifactId>
+    <artifactId>zerocode-rest-bdd</artifactId>
```

Comment:
Good spot. Happy to merge this PR now. We will update to `zerocode-tdd` later after we update the POM and get the tests run.

> however the `zerocode-tdd` is the latest artefact with that name.

FYI only-
+ zerocode-rest-bdd is till 1.2.17
+ zerocode-tdd is 1.3.0 onwards.